server_tests/test/clientTests.js

```diff
@@ -1322,13 +1323,13 @@ describe("RUST API TEST SUITE", function() {
             });
         });
 
-        it('Expect invalidParameterValue Error. camera._bublTimelapse expects active session\'s sessionId', function(done) {
```

Comment:
ok I've modified the test so that we can validate that the API has the right incorrect behaviour